main.js

```diff
@@ -21,6 +27,10 @@ function renderPhrase(arr) {
   $wholePhrase.classList.add('whole-string')
   for(var c = 0; c < arr.length; c++){
     $wholePhrase.appendChild(renderChar(arr[c]))
+    if (c === applicationState.currentIndex) {
+      let $temp = $wholePhrase.getElementsByTagName('span')[c]
+      $temp.classList.add('current-char')
```

Comment:
Instead of reaching into the `$wholePhrase`, make `renderChar` smart enough to return a correctly decorated character. `renderChar` is responsible for correctly rendering a character, so you shouldn't have to "fix" its return value.